tails_files/install.sh

```diff
@@ -1,17 +1,37 @@
 #!/bin/bash
+# SecureDrop persistent setup script for Tails 
 
 # check for root
 if [[ $EUID -ne 0 ]]; then
-    echo "This script must be run as root" 1>&2
-    exit 1
+	echo "This script must be run as root" 1>&2
```

Comment:
Please use consistent indentation throughout. You changed 2 spaces to 4 here, but later in the file there are instances of both 2 spaces and 4 spaces for indenting 1 level. Pick one (I prefer 2 spaces).